bin/www

```diff
@@ -12,7 +12,7 @@ var http = require('http');
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '8000');
+var port = normalizePort(process.env.PORT || '3000');
```

Comment:
it is again added in [raxerz's PR](https://github.com/systers/PC-Prep-Kit/pull/25/files)

please coordinate between yourselves to avoid redundancy, conflicts, & regressions